ShellScript/commands-builtin-shell-bash.yml

```diff
@@ -92,6 +92,7 @@ pwd:
 return:
   allow-end-of-options-token: false
   scope: keyword.control.return.shell
+         keyword.control.flow.return.shell
```

Comment:
Fun stuff. Did you run the Makefile in the ShellScript directory?

Reply:
Yes I did. Took a moment to figure out where things are coming from.